background/about-boilerplates.rst

```diff
@@ -0,0 +1,151 @@
+About Boilerplates
+==================
+
+As well as Django and Python applications, our Docker containerisation can
+include and build frontend components into your projects.
+
+Your project's *Boilerplate* will define the components - HTML, CSS and
+JavaScript - that are set up each time the project is built.
+
+Every Divio Cloud project includes a Boilerplate, whether it's one of our
+standard Boilerplates or a custom Boilerplate of your own, which will be
+selected when the project is created.
+
+
+Our built-in Boilerplates
+-------------------------
```

Comment:
there is also a Quickstart guide on our support area, dunno if we wanna link to it: http://support.divio.com/control-panel#boilerplates